python/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -269,6 +272,13 @@ def constructExtraFiles(self, job):
 
         folder_dir = master_job.getInputWorkspace(create=True).getPath()
 
+        extraopts = ' '
```

Comment:
Wouldn't this be better added to the `extraOpts` as the default value. This means that it's clearer to users if this has been added to the job description or not. Otherwise we have mystery lines sometimes added to jobs which may change the behavior in unexpected ways.